apps/backend/src/eventHandlers/MailService/EmailSettings.ts

```diff
@@ -1,4 +1,156 @@
-import { CreateTransmission } from 'sparkpost';
+interface BaseRecipient {
+  /** SparkPost Enterprise API only. Email to use for envelope FROM. */
+  return_path?: string | undefined;
+  /** Array of text labels associated with a recipient. */
+  tags?: string[] | undefined;
+  /** Key/value pairs associated with a recipient. */
+  metadata?: unknown;
+  /** Key/value pairs associated with a recipient that are provided to the substitution engine. */
+  substitution_data?: unknown;
+}
+
+interface MultichannelAddress {
+  /** The communication channel used to reach recipient. Valid values are “email”, “gcm”, “apns”. */
+  channel: string;
+  /** Valid email address. Required if channel is “email”. */
+  email: string;
+  /** User-friendly name for the email address. Used when channel is “email” */
+  name: string;
+  /** Email address to display in the “To” header instead of address.email (for BCC). Used when channel is “email” */
+  header_to: string;
+  /** SparkPost Enterprise API only. Required if channel is “gcm” or “apns” */
+  token: string;
+  /** SparkPost Enterprise API only. Required if channel is “gcm” or “apns” */
+  app_id: string;
+}
+
+interface Address {
+  /** Valid email address */
+  email: string;
+  /** User-friendly name for the email address */
+  name?: string | undefined;
```

Comment:
Is `?` and `undefined` redundant?? Is it necessary to have both?

Reply:
@martin-trajanovski

Reply:
You were right @yoganandaness. This is redundant as I copied the types from `sparkpost-node` package directly. Now it is cleaned up with the latest commit.